app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,65 @@
+module FastlyVCL
+  # Handles updates to our VCL snippet on Fastly that whitelists params
+  class WhitelistedParams
+    VCL_DELIMITER_START = "^(".freeze
+    VCL_DELIMITER_END = ")$".freeze
+    SNIPPET_NAME = "Whitelist certain querystring parameters".freeze
+    FILE_PARAMS = YAML.load_file("config/fastly/whitelisted_params.yml").sort.freeze
+
+    class << self
+      def update
+        fastly = Fastly.new(api_key: ApplicationConfig["FASTLY_API_KEY"])
+        service = fastly.get_service(ApplicationConfig["FASTLY_SERVICE_ID"])
+        latest_version = service.version
+
+        snippet = fastly.get_snippet(ApplicationConfig["FASTLY_SERVICE_ID"],
+                                     latest_version.number,
+                                     SNIPPET_NAME)
+
+        current_params = params_to_sorted_array(snippet.content)
```

Comment:
Relying on both the YAML file and these to be sorted leaves me feeling slightly uneasy. I see two alternatives:

1. Sets: turn both arrays into sets and use `Set#==`
2. Array differences: `(a - b).empty? && (b - a).empty?`

If we want to keep the sorting I at least add a comment to the YAML file.

Reply:
+1 on using a set

Reply:
I'd rather not use `Set` because then I'd have to turn it back into an `Array` to use `.join` when I'm logging the diffs to Datadog. I'll update the code to remove any reliance on params being ordered 👍